ospd/misc.py

```diff
@@ -408,3 +408,31 @@ def go_to_background():
     except OSError as errmsg:
         LOGGER.error('Fork failed: %s', errmsg)
         sys.exit(1)
+
+def create_pid(pidfile):
+    """ Check if there is an already running daemon and creates the pid file.
+    Otherwise gives an error. """
+
+    pid = str(os.getpid())
+
+    if os.path.isfile(pidfile):
```

Comment:
Please use [pathlib](https://docs.python.org/3/library/pathlib.html) instead
```suggestion
    if Path(pidfile).is_file():
```